Nishant/Day_12/postgres assignment/src/controller/student.ts

```diff
@@ -0,0 +1,90 @@
+import { Context } from "vm";
+
+import * as studentService from '../services/student';
+
+interface student_data{
+  student_id: number;
+  fname:string;
+  mname:string;
+  lname:string;
+  dob :Date;
+  gender: CharacterData;
+  address:string;
+  slice(a:number,b:number): student_data;
+}
+
+export async function getStudent(ctx: Context){
+    try{
+        let [rows]: Array<{rows: student_data}>=[];
+        let length:any;
+      rows=await  studentService.get_student();
```

Comment:
we shouldnt fetch everything from db and filter the result, can check how we can do pagination using db method, there limit and offset function in db.

Reply:
refer here https://github.com/adityajawade93/freshers-june2021/blob/e40de5ff0c6ecf17031c61a5c166caffb87aa0df/druval_cr/postgresAssignment/src/services/student.ts#L32